README.md

```diff
@@ -9,6 +9,11 @@ The Legend of Ericc is a turn-based single player rogue-like game made in Java w
 
 ## Log
 
+We decided to implement FOV and fog of war, this is the result (there are some animations that we need to add in later sprits),
+we based our fov algorithm on raytracing
+
+![Alt Text](https://media.giphy.com/media/9AIXNkGQFPCtiQTh1p/giphy.gif)
+
 After the first sprint we ended up with some stupid mobs (rendering is done real time, the mobs can turn only after the player makes a move, there are no mobs collision yet)
```

Comment:
może piszmy daty w tym naszym (b)logu? Bo first sprint itd. to zaraz przestanie być jasne co i kiedy się działo :p Nie chodzi mi o jakieś grube kminienie co kiedy powstało, tylko po prostu jak dodajemy nowy wpis tutaj to żeby był z datą. Also - można by założyć stronę na github.io i pisać tech bloga x)

Reply:
yup

Reply:
issue dodane